calico/getting-started/kubernetes/vpp/openshift.mdx

```diff
@@ -0,0 +1,211 @@
+curl -O "https://raw.githubusercontent.com/projectcalico/vpp-dataplane/{{vppbranch}}/yaml/platforms/openshift/03-role-calico-vpp-dataplane.yaml
+curl -O "https://raw.githubusercontent.com/projectcalico/vpp-dataplane/{{vppbranch}}/yaml/platforms/openshift/03-rolebinding-calico-vpp-dataplane.yaml
+curl -O "https://raw.githubusercontent.com/projectcalico/vpp-dataplane/{{vppbranch}}/yaml/platforms/openshift/03-serviceaccount-calico-vpp-dataplane.yaml
+curl -O "https://raw.githubusercontent.com/projectcalico/vpp-dataplane/{{vppbranch}}/yaml/platforms/openshift/04-calico-vpp-nohuge.yaml
+cd ..
+cp vpp/* manifests/
+```
+
+Make sure that SERVICE_PREFIX in `manifests/03-configmap-calico-vpp-resources.yaml` matches the service cidr of the cluster:
+
+```bash
+SERVICE_CIDR=`grep -A1 serviceNetwork: ./manifests/cluster-config.yaml | tail -n1 | cut -d '-' -f2`
+sed -i "s#SERVICE_PREFIX:.*#SERVICE_PREFIX:$SERVICE_CIDR#" ./manifests/03-configmap-calico-vpp-resources.yaml
+```
+
+
+### Optionally provide additional configuration
+
+You may want to provide Calico or the VPP dataplane with additional configuration at install time. For example, BGP configuration or peers.
+You can use a Kubernetes ConfigMap with your desired Calico resources in order to set configuration as part of the installation.
+If you do not need to provide additional configuration, you can skip this section.
+
+To include [Calico resources](../../../reference/resources/index.mdx) during installation, edit `manifests/02-configmap-calico-resources.yaml` in order to add your own configuration.
+
+In order to customize the configuration of the VPP dataplane, edit `manifests/03-configmap-calico-vpp-resources.yaml`. For details refer to the **Configuration options** section of [Getting Started](./getting-started.mdx).
+
+:::note
+
+If you have a directory with the Calico resources, you can create the file with the command:
+
+```
+oc create configmap -n tigera-operator calico-resources \
+--from-file=<resource-directory> --dry-run -o yaml \
+manifests/02-configmap-calico-resources.yaml
+```
+
+With recent versions of `oc` it is necessary to have a kubeconfig configured or add `--server='127.0.0.1:443'`
+even though it is not used.
+
+:::
+
+:::note
+
+If you have provided a `calico-resources` ConfigMap and the tigera-operator pod fails to come up with `Init:CrashLoopBackOff`,
+check the output of the init-container with `oc logs -n tigera-operator -l k8s-app=tigera-operator -c create-initial-resources`.
+
+:::
+
+### Create the cluster
+
+Start the cluster creation with the following command and wait for it to complete.
+
+```bash
+openshift-install create cluster
+```
+
+Once the above command is complete, you can verify {{prodname}} VPP is installed by verifying the components are available with the following command.
+
+```
+oc get tigerastatus
+```
+
+:::note
+
+To get more information, add `-o yaml` to the above command.
+
+:::
```

Comment:
remove

Reply:
Done. Btw, this would also be applicable to the **Calico openshift** doc.